crm_compassion/views/interaction_resume_view.xml

```diff
@@ -57,6 +59,7 @@
                 <group expand="0" string="Group By...">
                     <filter name="Type" domain="[]" context="{'group_by':'communication_type'}"/>
                     <filter name="Direction" domain="[]" context="{'group_by':'direction'}"/>
+                    <filter name="communication_date" domain="[]" context="{'group_by':'communication_date'}"/>
```

Comment:
I just noticed that filter `name` attribute is a technical name and you should add a `string` attribute with human readable name (for the user interface). Can you as well add the `string` attribute in the other filters?